app/models/user.rb

```diff
@@ -1,6 +1,12 @@
 class User < ApplicationRecord
   include Mergeable
 
+  SYNC_STATUSES = {
```

Comment:
Can we simplify this and make it an `enum` instead?

Reply:
Enums convert the value into an int column and the value is order specific. We can use a hash to define the Enum instead, in which case the order does not matter. 
Eg: `enum sync_approval_status: { waiting: 0, approved: 2, disapproved: 1}` 

But converting the value to int makes it rails specific. Someone looking at the DB will not directly know what the status is. 

We can define the enum as 
```enum sync_approval_status {
 waiting: 'waiting_for_approval', 
disapproved: 'disapproved_for_syncing',
approved: 'approved_for_syncing',
```
to store strings in the database.
I've tested this and it works, but the behavior is undocumented.